src/Visualization/Python/Render3D/RenderBbh.py

```diff
@@ -0,0 +1,257 @@
+
+    # Slice volume data
+    slice = pv.Slice(registrationName="slice", Input=volume_files_xmf)
+    slice.SliceType = "Plane"
+    slice.HyperTreeGridSlicer = "Plane"
+    slice.SliceOffsetValues = [0.0]
+    slice.SliceType.Normal = [0.0, 0.0, 1.0]
+    slice.Triangulatetheslice = 0
+
+    # Warp grid by spatial ricci scalar
+    warp_by_scalar = pv.WarpByScalar(
+        registrationName="WarpByScalar", Input=slice
+    )
+    warp_by_scalar.Scalars = ["POINTS", "SpatialRicciScalar"]
+    warp_by_scalar.ScaleFactor = 2.5
+    warp_by_scalar.Normal = [0.0, 0.0, -1.0]
+    warp_by_scalar_display = pv.Show(
+        warp_by_scalar, render_view, "GeometryRepresentation"
+    )
+    warp_by_scalar_display.SetScalarBarVisibility(render_view, False)
+
+    # Apparent Horizon Visualization
+    if aha_xmf:
+        ah_vis(aha_xmf, render_view)
+    if ahb_xmf:
+        ah_vis(ahb_xmf, render_view)
+
+    if show_grid:
+        warp_by_scalar_display.Representation = "Surface With Edges"
+
+    pv.LoadPalette(paletteName="GradientBackground")
+    render_view.OrientationAxesVisibility = 0
+    pv.SetActiveSource(warp_by_scalar)
+    warp_by_scalar_display.Opacity = 0.8
+    pv.ColorBy(warp_by_scalar_display, ("POINTS", "Lapse"))
+    layout = pv.GetLayout()
+    layout.SetSize(1920, 1080)
+
+    # Camera placements
+    # Top down view
+    if camera_angle == "Top":
+        render_view.CameraPosition = [0.0, 0.0, 36.90869716569761]
+        render_view.CameraFocalPoint = [0.0, 0.0, 0.6894899550131899]
+        render_view.CameraViewUp = [0, 1, 0]
+        render_view.CameraParallelScale = 424.27024700303446
+    # Wide/Inbetween View
+    elif camera_angle == "Wide":
+        render_view.CameraPosition = [
+            -89.0,
+            -17.0,
+            25.0,
+        ]
+        render_view.CameraFocalPoint = [
+            -0.3921962951264054,
+            1.6346750682876983,
+            -0.34522248814953405,
+        ]
+        render_view.CameraViewUp = [
+            0.0,
+            0.0,
+            1.0,
+        ]
+    # Side View
+    else:
+        render_view.CameraPosition = [
+            -29.944619336722987,
+            -3.666072157343372,
+            2.895224044348878,
+        ]
+        render_view.CameraFocalPoint = [
+            -0.13267040638072278,
+            0.6356115665206243,
+            -0.37352608789235847,
+        ]
+        render_view.CameraViewUp = [0.0, 0.0, 1.0]
+        render_view.CameraParallelScale = 519.6152422706632
+
+    # Simulation time
+    if show_time:
+        time_filter = pv.AnnotateTimeFilter(
+            registrationName="annotate_time_filter", Input=slice
+        )
+        time_filter.Format = "Time: {time:0.2f}M"
+        annotate_time_filter_display = pv.Show(
+            time_filter, render_view, "TextSourceRepresentation"
+        )
+        annotate_time_filter_display.FontSize = 45
+
+    # Save animation
+    pv.SaveAnimation(output, render_view)
```

Comment:
Can you do the same selection between single time step and animation here as is done in the other Render3d scripts?